README.md

```diff
@@ -1 +1,109 @@
-# capstone-1
\ No newline at end of file
+<a name="readme-top">Fitness</a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 'Fitness <a name="about-project"></a>
+
+**Fitness Website** is a project based on first mobile structure and it is complete static website with even desktop responsive version.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Website for fitness seminar</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project you need: only to clone this github repo
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- cd my-folder
+- git clone https://github.com/husush23/capstone-1.git
+
+### Install
+
+no installation needed, just clone is enough
+
+### Deployment
+
+You can deploy this project using any hosting server in the world
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors">Hussein Hashi</a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/husush23)
+- Twitter: [@twitterhandle](https://twitter.com/HusseinKadare2)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/hussein-kadare-8aa1361a8/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Full website**
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please follow on my social media platform listed above.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- The owner of the original design is Cindy Shin in Behance.
+
+- You see video describing this website here: https://www.loom.com/share/1a7ec921c5fd4d5b8630b9365f442f1e
```

Comment:
- **[OPTIONAL]** ⚠️ **IMPORTANT** ⚠️ The video link should have been added in the PR description, take that in mid for the next capstone project 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/234667850-e06e5aa8-c21c-4b74-8f0f-57602dd1cd1f.png)